src/coffee/client.coffee

```diff
@@ -220,9 +226,11 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    if options.enableRepeater? and !options.enableRepeater
```

Comment:
@PhilippSpo, as @butenkor [already mentioned](https://github.com/sphereio/sphere-node-sdk/pull/190#discussion_r92011601), the idea is to turn the repeater on if user of SDK will not send any flag to it. So it's not a side-effect, but a feature.